NEWS.md

```diff
@@ -175,6 +175,9 @@
 
 *  SQL translation for Apache Impala (@edgararuiz)
 
+*  SQL translation for Oracle  (@edgararuiz)
```

Comment:
Can you please move up under `1.0.0.9000`. You should also mention the analyse support there too.